Automattic/wp-calypso
# Migration: Add migration assistance step and zendesk ticket creation

## Proposed Changes

Context: pdDR7T-1uw-p2

This PR adds a new step to the `import-hosted-site` (tbd in which flow we need this?), that appears after the checkout page when a user accepts a free migration assistance.

![image](https://github.com/Automattic/wp-calypso/assets/52076348/b2d0901b-8076-4fe0-93a6-68b86839f3fe)

When the user reaches this step, we create a new Zendesk ticket for HEs to handle the migration assistance.

## Testing Instructions
- Checkout this diff: D146818-code
- Ensure that you are sandboxed before testing (public API)
- Follow the flow '/setup/import-hosted-site' and create a site
- Visit /setup/import-hosted-site/migrateMessage?siteSlug=yoursite.wordpress.com
- Checkout if a new ticket is created at the zendesk sandbox
- In the patch where the mock support-activity is added change it to match your site: 'Migration request yoursite.wordpress.com'
- Visit /setup/import-hosted-site/migrateMessage?siteSlug=yoursite.wordpress.com again
- No new ticket should be created this time.
